### PR TITLE
Support EtherCore

### PR DESCRIPTION
**Pull request by ethcoreorg**
## Description
Support **EtherCore Mainnet**, Proof of Work Ethereum compatible network.

Homepage: https://ethercore.org
Block Explorer: https://explorer.ethercore.org
RPC Endpoint: https://rpc.ethercore.org
ChainId / SLIP-44: **466** https://github.com/satoshilabs/slips/pull/847
Official Email: info@ethercore.org

ps) We use ProgPoW 0.9.3 version as suggested on Specification https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1057.md#specification - 0.9.2 version is considered as deprecated since it wouldn't benefit mining with AMD cards.

### CODE REVIEW

**Comment by ethcoreorg** [comment]
[clabot:check]

File: ethcore/res/ethereum/ethercore.json [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+							}
+						}
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000009": {
+			"builtin": {
+				"name": "blake2_f",
+				"activate_at": "0x0",
```
Comment (niklasad1) [inline ethcore/res/ethereum/ethercore.json]: Rationale why `blake2_f` is activated at 0 and `alt_bn128_xx` cost reductions at `0xbe10b`?

Do you follow the EIPs activated on Istanbul or something else?

Comment (ethcoreorg) [reply ethcore/res/ethereum/ethercore.json]: Nice catch, thank you!

Comment (ordian) [reply ethcore/res/ethereum/ethercore.json]: ah, I wonder if there is a tool to warn you about potential errors of copy-pasting code :)

**Comment by ethcoreorg** [comment]
@niklasad1 Addressed comments!

**Comment by ethcoreorg** [comment]
@niklasad1 Addressed comments!

**Comment by ethcoreorg** [comment]
Builtin now match with foundation and ropsten specs https://github.com/paritytech/parity-ethereum/blob/master/ethcore/res/ethereum/ropsten.json#L3328

**Pull request merged**